src/pages/cash-dtd/form/CaDocumentTypeDefaultForm.js

```diff
@@ -0,0 +1,208 @@
+    })()
+  }, [])
+
+  return (
+    <FormShell resourceId={ResourceIds.CaDtd} form={formik} maxAccess={maxAccess} editMode={editMode} isCleared={false}>
+      <VertLayout>
+        <Grow>
+          <Grid container spacing={4}>
+            <Grid item xs={12}>
+              <ResourceComboBox
+                endpointId={SystemRepository.DocumentType.qry}
+                parameters={`_startAt=0&_pageSize=1000&_dgId=${3303}`}
+                name='dtId'
+                required
+                label={labels.documentType}
+                columnsInDropDown={[
+                  { key: 'reference', value: 'Reference' },
+                  { key: 'name', value: 'Name' }
+                ]}
+                valueField='recordId'
+                displayField={['reference', 'name']}
+                readOnly={editMode}
+                values={formik.values}
+                maxAccess={maxAccess}
+                onChange={(event, newValue) => {
+                  formik && formik.setFieldValue('dtId', newValue?.recordId || '')
+                }}
+                error={formik.touched.dtId && Boolean(formik.errors.dtId)}
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <ResourceComboBox
+                endpointId={SystemRepository.Plant.qry}
+                name='plantId'
+                label={platformLabels.plant}
+                valueField='recordId'
+                displayField={['reference', 'name']}
+                columnsInDropDown={[
+                  { key: 'reference', value: 'plant Ref' },
+                  { key: 'name', value: 'Name' }
+                ]}
+                values={formik.values}
+                onChange={(event, newValue) => {
+                  formik.setFieldValue('plantId', newValue?.recordId || '')
+                }}
+                error={formik.touched.plantId && Boolean(formik.errors.plantId)}
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <ResourceLookup
+                endpointId={CashBankRepository.CashAccount.snapshot}
+                parameters={{
+                  _type: 0
+                }}
+                name='cashAccountId'
+                label={labels.cA}
+                valueField='reference'
+                displayField='name'
+                valueShow='cashAccountRef'
+                secondValueShow='cashAccountName'
+                form={formik}
+                onChange={(event, newValue) => {
+                  formik.setFieldValue('cashAccountId', newValue?.recordId || '')
+                  formik.setFieldValue('cashAccountRef', newValue?.reference || '')
+                  formik.setFieldValue('cashAccountName', newValue?.name || '')
+                }}
+                error={formik.touched.cashAccountId && Boolean(formik.errors.cashAccountId)}
+                maxAccess={maxAccess}
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <ResourceLookup
```

Comment:
do we need cashAccount, fromCashAccount and toCashAccount ? I thought only from and to